subsys/bluetooth/audio/Kconfig.otc

```diff
@@ -0,0 +1,34 @@
+# Bluetooth Audio - Object transfer configuration options
```

Comment:
I guess why this was moved to the `audio` directory, but can't we just as easily put it next to OTS?

Reply:
This comment applies for the entirety of OTC (header filers, Kconfigs, etc.)

Reply:
The goal is to put it there.  But I would like to keep the OTC "hidden" here until we have cleaned it up and brought it up to the standard of the OTS.